web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js

```diff
@@ -4,192 +4,151 @@ import { Description } from '../../../components/Description/Description';
-        To import predictions without connecting a model,
-        {" "}
-        <a href="https://labelstud.io/guide/predictions.html" target="_blank">
-          see the documentation
-        </a>.
-      </Description>
-      <Button onClick={() => showMLFormModal()}>
-        Add Model
-      </Button>
-
-      <Divider height={32}/>
-
-      <Form action="updateProject"
-        formData={{ ...project }}
-        params={{ pk: project.id }}
-        onSubmit={() => fetchProject()}
-        autosubmit
-      >
-        <Form.Row columnCount={1}>
-          <Label text="ML-Assisted Labeling" large/>
-
-          <div style={{ paddingLeft: 16 }}>
-            <Toggle
-              label="Start model training after any annotations are submitted or updated"
-              name="start_training_on_annotation_update"
-            />
-          </div>
-
-          <div style={{ paddingLeft: 16 }}>
-            <Toggle
-              label="Retrieve predictions when loading a task automatically"
-              name="evaluate_predictions_automatically"
-            />
-          </div>
-
-          <div style={{ paddingLeft: 16 }}>
-            <Toggle
-              label="Show predictions to annotators in the Label Stream and Quick View"
-              name="show_collab_predictions"
-            />
-          </div>
-        </Form.Row>
-
-        {(!isFF(FF_DEV_1682) || !backends.length ) && (
-          <ProjectModelVersionSelector />
-        )}
-
-        <Form.Actions>
-          <Form.Indicator>
-            <span case="success">Saved!</span>
-          </Form.Indicator>
-          <Button type="submit" look="primary" style={{ width: 120 }}>Save</Button>
-        </Form.Actions>
-      </Form>
-
-      <MachineLearningList
-        onEdit={(backend) => showMLFormModal(backend)}
-        fetchBackends={fetchBackends}
-        backends={backends}
-      />
-    </>
-  );
+export const MachineLearningSettings = () => {
+    const api = useAPI();
+    const { project, fetchProject } = useContext(ProjectContext);    
+    const [backends, setBackends] = useState([]);    
+    const [loading, setLoading] = useState(false);
+    const [loaded, setLoaded] = useState(false);
+  
+    const fetchBackends = useCallback(async () => {
+        setLoading(true);
+        const models = await api.callApi('mlBackends', {
+            params: {
+                project: project.id,
+                include_static: true
+            },
+        });
+        
+        if (models) setBackends(models);
+        setLoading(false);
+        setLoaded(true);
+    }, [project, setBackends]);
+
+    const startTrainingModal = useCallback((backend) => {
+        const modalProps = {
+            title: `Start Model Training`,
+            style: { width: 760 },
+            closeOnClickOutside: true,
+            body: <StartModelTraining backend={backend} />
+        };
+        
+        const modalRef = modal(modalProps);
```

Comment:
This doesn't appear to be referenced within scope, so we can drop it down to:
```suggestion
        modal(modalProps);
```